novoed/api_test.py

```diff
@@ -41,15 +42,29 @@ def patched_post(mocker):
 
 @pytest.mark.django_db
 @pytest.mark.parametrize(
-    "response_status,exp_created,exp_existing",
-    [[status.HTTP_200_OK, True, False], [status.HTTP_207_MULTI_STATUS, False, True]],
+    "response_status,exp_created,exp_existing,exp_update_sync_date",
+    [
+        [status.HTTP_200_OK, True, False, True],
+        [status.HTTP_207_MULTI_STATUS, False, True, True],
+        [status.HTTP_204_NO_CONTENT, False, False, False],
+    ],
 )
 def test_enroll_in_novoed_course(
-    patched_post, novoed_user, response_status, exp_created, exp_existing
-):
+    patched_post,
+    novoed_user,
+    response_status,
+    exp_created,
+    exp_existing,
+    exp_update_sync_date,
+):  # pylint:disable=too-many-arguments
     """
     enroll_in_novoed_course should make a request to enroll a user in NovoEd and return flags indicating the results
     """
+    enrollment = BootcampRunEnrollmentFactory.create(
+        user=novoed_user,
+        bootcamp_run__novoed_course_stub=FAKE_COURSE_STUB,
+        novoed_sync_date=None,
```

Comment:
Is it worth adding a test where `novoed_sync_date` is not `None`, to make sure it doesn't get updated?

Reply:
Added (sorry, forgot about this PR!)